base/client.jl

```diff
@@ -211,6 +211,20 @@ end
 # call include() on a file, ignoring if not found
 include_ifexists(mod::Module, path::AbstractString) = isfile(path) && include(mod, path)
 
+function hascolorTTY()
+    term_type = get(ENV, "TERM", @static Sys.iswindows() ? "" : "dumb")
```

Comment:
Seems like the Windows check isn't doing anything meaningful here; perhaps this could just be
```suggestion
    term_type = get(ENV, "TERM", "")
```
?